src/main/resources/assets/computercraft/lua/rom/programs/echo.lua

```diff
@@ -0,0 +1 @@
+print( ... )
```

Comment:
In the PR you're liked we don't have Vars in the Shell. In this PR we have Vars in the Shell.